assets/js/components/Link.js

```diff
@@ -90,6 +110,7 @@ Link.propTypes = {
 	caps: PropTypes.bool,
 	danger: PropTypes.bool,
 	disabled: PropTypes.bool,
+	ariaLabel: PropTypes.string,
```

Comment:
I think we can remove this rather than rename it since it isn't really a custom prop but an HTML attribute we pass through.
```suggestion
```